services/worker-manager/test/fakes/schemas/google-instance.yml

```diff
@@ -0,0 +1,89 @@
+# This is a partial schema for google.compute.instances.insert.  Feel free to add additional
+# properties here if necessary, consulting with the relevant bits of Azure documentation.
+#
+# NOTE: the `testProperty` properties here are used in the tests to verify that
+# additional configuration from the user is correctly passed to the endpoint.
+#
+# https://cloud.google.com/compute/docs/reference/rest/v1/instances/insert
+type: object
+definitions:
+  # https://cloud.google.com/resource-manager/docs/creating-managing-labels
+  labels:
+    type: object
+    propertyNames:
+      pattern: '^[a-z0-9_-]*$'
```

Comment:
Is this the thing with the confusing set of allowed chars we talked about yesterday?

So this is the things they accept but without the things that we don't allow? (non-ascii, etc)

Reply:
Yeah, I figure since we're not injecting any unicode in the tests, this is enough.  It will catch anything where we accidentally include a `/` for example.